src/pages/interviewers.tsx

```diff
@@ -70,6 +78,10 @@ function Row({ user, interviews }: {
       </Tag>}
     </Td>
     <Td>{interviews}</Td>
+    <Td>
+      {noMoreThan !== undefined && until !== undefined 
+        ? `${noMoreThan} 直到 ${until.slice(0, 10)}` : ''}
```

Comment:
why not simply `&& ...}` and remove `: ''`

Reply:
that would remove entire column if no one sets limit.